Jonah/stylesheet.css

```diff
@@ -0,0 +1,21 @@
+
```

Comment:
Great job on the CSS Jonah! Lets clean up some of the indentation and unnecessary white space and then it looks good!